docker/Dockerfile-python

```diff
@@ -10,4 +10,4 @@ RUN pip install python-ldap pytest
 
 WORKDIR /work
 
-CMD ["py.test", "test.py"]
+CMD ["py.test", "test.py","-s"]
```

Comment:
What is the reason for this? People running tests usually expect default capture behavior.

Reply:
It was to see prints. It will be removed.

Reply:
I see. If you execute `py.test` with `--pdb`, it will even start a debugger if there is a test failure - may be good to know!